app/controllers/sign_up/select_email_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module SignUp
+  class SelectEmailController < ApplicationController
```

Comment:
While it made testing a little easier, I probably shouldn't be able to go straight to http://localhost:3000/sign_up/select_email if I'm signed-in. We should probably have something similar to CompletionController's [`verify_needs_completion_screen`](https://github.com/18F/identity-idp/blob/b8a3933bc536590a84a0e1c459445a4b9ee0f71c/app/controllers/sign_up/completions_controller.rb#L10) to ensure that I can only access it if I'm in the completions part of the flow.